scripts/configure.py

```diff
@@ -113,18 +113,35 @@
         ]
     },
     'logging': {
+        "copy": [
+          {
+            'from': 'config/deactivate_persist_logs_crontab',
+            'to': '/etc/cron.d/deactivate_persist_logs',
+            'sudo': 'true',
+          },
+          {
+            'from': 'scripts/increment_auto_off.py',
+            'to': '/usr/local/bin/increment_auto_off.py',
+            'sudo': 'true',
+          },
+        ],
         'script': [
             'echo "reconfiguring secondary logging utility rsyslog to only allow remote logging"',
             f"echo '{RSYSLOG_CFG}' | sudo tee /etc/rsyslog.conf",
             # just in case it was disabled...
             'sudo systemctl enable rsyslog.service',
             'sudo systemctl restart rsyslog.service',
 
-            'echo "reconfiguring journald to only log to RAM"',
-            r'echo -e "[Journal]\nStorage=volatile\nRuntimeMaxUse=64M\nForwardToConsole=no\nForwardToWall=no\n" | sudo tee /etc/systemd/journald.conf',
+            'echo "If first deploy, reconfiguring journald to only log to RAM"',
+            r'[ ! -d /var/log/journal ] && echo -e "[Journal]\nStorage=volatile\nRuntimeMaxUse=64M\nForwardToConsole=no\nForwardToWall=no\n" | sudo tee /etc/systemd/journald.conf',
             'sudo systemctl enable systemd-journald.service',
             'sudo systemctl restart systemd-journald.service',
 
+            'echo Handle dependencies for log persistence options',
+            'sudo mkdir -p /var/log/journal',
```

Comment:
> I've set persistence, and I see /etc/systemd/journald/conf reflect storage=persistent. When I run another update afterwards, it resets to storage=volatile, with no regard for my setting.

Not true, this line of `configure.py` checks if the `/var/log/journal` directory exists, and if not it sets the `journald.conf` file to our default
`/var/log/journal` does not exist by default, but is created a few lines down so this functionally works as an "Only on first run of `configure.py`, set `journald.conf` to these parameters"